assets/css/global.css

```diff
@@ -48,24 +49,30 @@ nav {
 /* hero */
 header {
     min-height: 100vh;
-    background: linear-gradient(rgba(255, 0, 0, 0.200), rgba(0, 128, 0, 0.200)),
-      url(../images/hero-test.webp) center/cover no-repeat;
+    background: linear-gradient(rgba(255, 0, 0, 0.455), rgba(0, 128, 0, 0.455)),
```

Comment:
can index page specific styles be moved to a index specific css file and be added to index.html instead of being added to global stylesheet?

Reply:
Sure I'll move all specific styles